deployments/anvilbox/environment.py

```diff
@@ -141,6 +141,7 @@ def env() -> Mapping[str, Optional[str]]:
         'AZUL_TDR_SOURCE_LOCATION': 'us-central1',
         'AZUL_TDR_SERVICE_URL': 'https://jade.datarepo-dev.broadinstitute.org',
         'AZUL_SAM_SERVICE_URL': 'https://sam.dsde-dev.broadinstitute.org',
+        'AZUL_DUOS_SERVICE_URL': 'https://consent.dsde-dev.broadinstitute.org',
```

Comment:
We need an declaration in the root `environment.py` with documentation and a default of `None`. The default value should prevent the feature altogether. That way we can deploy this to `hammerbox` and `anvilprod` without breaking them.